### ISSUE TITLE
Docs need review: not following best practices and style guide yet

### ISSUE THREAD

**Issue by svx** [collective/collective.solr#67]
Hi,

before we can include them into docs.plone.org, we have to review the, a 'quick' review showed that currently the docs are not following best practices and our style guide in terms of *rst syntax*, semantic linefeeds and some more stuff :)

I say this is a perfect topic to sprint :)

**Comment by gforcada** [comment]
@loechel any progress so far?

**Comment by svx** [comment]
Nope, no one was working on that and that is one of the reasons why it is still not merged

**Comment by loechel** [comment]
Sorry to much to do and didn't find the time. Maybe end of december or anyone else is willing to do so?
